### DISCUSSION TITLE
Callsign double

### DISCUSSION THREAD

**Discussion by MStallen** [flybywiresim/aircraft#6320] [Help]
Without loading simbrief my callsign with MSFS 2020 is K-L-M 301
After pressing init on the MCDU it's K-L-M kilo mike lima 301

In simbrief : 
- Airline     : KLM
- FlightNo : 301
- ATC Callsign : KLM301

In MSFS
- Callsign  : KLM
- FlightNo: 301

How to change so it stays K-L-M 301 ?

**Comment by vsbrt** [comment]
change the "FLT NBR" in MCDU manually after  pressing INIT REQ. So after INIT REQ MCDU will show KLM301 in FLT NBR field similar to the way its shown in simbrief. ommit KLM and then just enter the flight number 301 in this case  into scratch pad and then click on corresponding button beside FLT NBR on MCDU. This way MSFS ATC will use K-L-M 301.

**Reply by Midnightmusic** [reply]
You don't have to do that if the entries are correct in the aircraft.cfg, you just put in the flight number and ATC will call the correct terminology. The example below shows my entry in the aircraft.cfg

atc_airline = "KLM" ; airline name
atc_flight_number = "301" ; flight number
